src/cartesian/ZAxis.tsx

```diff
@@ -20,9 +21,7 @@ export interface Props {
   domain?: AxisDomain;
 }
 
-function ZAxis(props: Props): any {
```

Comment:
In general, the types should be safer now. The `FunctionComponent` return type is not defined as `any`, but rather all the types that are valid as a React component.